pages/profilepage.html

```diff
@@ -16,33 +16,48 @@
     rel="stylesheet"
     href="https://unpkg.com/swiper@7/swiper-bundle.min.css"
     />
+    <script src="http://code.jquery.com/jquery-1.11.0.min.js"></script>
+
     <script type="text/javascript" defer src="../assets/vendor/js/profile_page.js"></script>
     <title>Profile-Page</title>
 </head>
+<script>
+function profile(){
+$.post("../../backend/profile.php", {
+				id:$_SESSION['userid'],
```

Comment:
using PHP "$_SESSION" on this html file may not be very proper. Do you actually need it? I can't also find the file called "backend/profile.php"